controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -300,6 +301,77 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) error {
 		}
 		// Else: nothing for eBPF at the moment
 	}
+
+	// Connection tracking stage
+	endTimeout, err := time.ParseDuration(b.desired.Processor.ConnectionEndTimeout)
+	if err != nil {
+		return err
+	}
+	updateInterval, err := time.ParseDuration(b.desired.Processor.ConnectionUpdateInterval)
+	if err != nil {
+		return err
+	}
```

Comment:
I think this is where we need to be careful: even if there's a default duration provided in the CRD, we must assume, after an upgrade (and if we still use v1alpha1), those values could be empty strings. I'm not 100% sure of that though

Reply:
cc @OlivierCazade @msherif1234

Reply:
These should not at this level since it's supposed to be converted by `Convert_v1alpha1_FlowCollector_To_v1beta1_FlowCollector` function

https://github.com/netobserv/network-observability-operator/pull/258/files#diff-23e9a27bd609a9414ced41fdaf8063ade03a51b3154d85ad4edfdae228dd5531